emily_cron/app/clients/mempool.py

```diff
@@ -7,11 +7,48 @@
 logger = logging.getLogger(__name__)
 
 
+def _collect_rbf_txids(data: dict[str, Any] | None) -> set[str]:
```

Comment:
nit: Since the RBF transactions form a chain, a list or an ordered set might be more appropriate.